plugin.py

```diff
@@ -36,15 +36,28 @@ def is_allowed_to_start(
         # Config pattern is found here:
         # https://github.com/tailwindlabs/tailwindcss-intellisense/blob/766a5d533dcb68640ce6b3270488f6701dd1173d/packages/vscode-tailwindcss/src/extension.ts#L40
         config_file_pattern = r'^(tailwind|tailwind\.config)\.(js|cjs)$'
-        folder_exclude_patterns = set(sublime.load_settings('Preferences.sublime-settings').get("folder_exclude_patterns", [])) # type: Set[str]
-        folder_exclude_patterns.add('node_modules') # definitely exclude node_modules
+        folder_exclude_patterns = set(
+            sublime.load_settings('Preferences.sublime-settings').get("folder_exclude_patterns", [])
+        )  # type: Set[str]
+        folder_exclude_patterns.add('node_modules')  # definitely exclude node_modules
         config_file = find_file_in_workspace(config_file_pattern, workspace_folders[0].path, folder_exclude_patterns)
-        if config_file:
-            return None # config found, return None to start the session
-        return "No tailwind configuration file present in the workspace folder."
+        if not config_file:
+            return "No tailwind configuration file present in the workspace folder."
+        if not LspTailwindcssPlugin.is_tailwind_installed(workspace_folders[0].path):
+            return "`tailwindcss` dependency is not installed. Run `npm install` in the workspace to enable LSP-tailwindcss."
```

Comment:
Should it say "Run `npm install tailwindcss`" instead?

Reply:
Or maybe even some more generic wording as some use "yarn", some use "pnpm", etc.

Reply:
What do you think about:
"`tailwindcss` dependency is not installed in the workspace."

it will show the following in the status bar.
`Cannot start LSP-tailwindcss: 'tailwindcss' dependency is not installed in the workspace.`

Reply:
For now I will put this generic message from the comment above.

If you have a better error message, I am all ears :)